app/services/save_offer_and_conditions.rb

```diff
@@ -0,0 +1,13 @@
+class SaveOfferAndConditions
+  attr_reader :application_choice, :conditions
+
+  def initialize(application_choice:, conditions:)
+    @application_choice = application_choice
+    @conditions = conditions
+  end
+
+  def save
+    application_choice.offer = { 'conditions' => conditions }
```

Comment:
Once the read PR is done, I think we can move this line into UpdateOfferConditions and not have this service at all